MKLDNN_README.md

```diff
@@ -0,0 +1,287 @@
+# Build/Install MXNet with MKL-DNN
+
+<h2 id="0">Contents</h2>
+
+* [1. Linux](#1)
+* [2. MacOS](#2)
+* [3. Windows](#3)
+* [4. Verify MXNet with python](#4)
+* [5. Enable MKL BLAS](#5)
+
+<h2 id="1">Linux</h2>
+
+### Prerequisites
+
+```
+apt-get update && apt-get install -y build-essential git libopencv-dev curl gcc libopenblas-dev python python-pip python-dev python-opencv graphviz python-scipy python-sklearn

```

Comment:
Can you format so it is easier to read? Maybe add sudo since most people would need that (unless this is intended to be docker instructions).
```
sudo apt-get update && sudo apt-get install -y \
build-essential \
curl \
... alphabetical list
```

Reply:
keep consistent with https://mxnet.incubator.apache.org/install/windows_setup.html